tests/Makefile.am

```diff
@@ -227,6 +227,12 @@ EXTRA_DIST = \
 	_test_volatilestate \
 	_test_wrongorder
 
+testdir = $(libexecdir)/installed-tests/$(PACKAGE)
+
+install-data-hook:
+	mkdir -p $(DESTDIR)$(testdir)
+	cd $(srcdir) && cp --parents $(EXTRA_DIST) $(DESTDIR)$(testdir)
```

Comment:
`make distcheck` doesn't complete as normal user anymore...

Reply:
Fun, `cp --parents` creates the directory for the first file and then sets the directory read-only, like the source. Other file copy fails :) fixed